src/stubs/LocalizedVagrantfile

```diff
@@ -0,0 +1,23 @@
+require 'json'
+require 'yaml'
+
+VAGRANTFILE_API_VERSION = "2"
+confDir = $confDir ||= File.expand_path("~/.composer/vendor/laravel/homestead")
```

Comment:
Why would we use ~/.composer here? Wouldn't we just use the project's composer/vendor/ directory?